src/lava/proc/sdn/process.py

```diff
@@ -13,14 +14,14 @@
 @unique
 class ActivationMode(IntEnum):
     """Enum for synapse sigma delta activation mode. Options are
-    {``UNIT : 0``, ``RELU : 1``}.
+    UNIT: 0
```

Comment:
Remove the redundant listing of the options in the docstring.

Reply:
What do you mean?